testframes/test_forward_solution/test_forward_solution.cpp

```diff
@@ -177,6 +176,11 @@ void TestForwardSolution::compareForward()
 
     printf(">>>>>>>>>>>>>>>>>>>>>>>>> Compare Forward Solution >>>>>>>>>>>>>>>>>>>>>>>>>\n");
 
+    QString refFwdFileName(QDir::currentPath()+"./mne-cpp-test-data/Result/sample_audvis-meg-oct-6-fwd.fif");
+
+    //Load data
+    MNEForwardSolution t_Fwd(refFwdFileName);
```

Comment:
MNEForwardSolution constructor is not overloaded for Qstring. You probably forgot to create a QFile first.

Reply:
Thank you - yes indeed!

Reply:
Thank you - yes indeed!